src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/RunnableProjectTemplate.cs

```diff
@@ -23,6 +24,7 @@ public RunnableProjectTemplate(JObject raw, IGenerator generator, IFile configFi
             ShortName = config.ShortName;
             Author = config.Author;
             Tags = config.Tags;
+            CacheParameters = config.CacheParameters;
             Description = config.Description;
```

Comment:
Since we assert that `Tags` and `CacheParameters` are non-null by iterating them lower in the method (and these are get-only) we should probably guarantee they're non null in the ctor